src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The stylesheet link should point to a CSS file, not an SCSS file. Please change the href to 'styles/main.css' to ensure the correct file is linked.